prelude/cxx/tools/BUCK.v2

```diff
@@ -22,15 +22,27 @@ prelude.command_alias(
     visibility = ["PUBLIC"],
 )
 
+prelude.export_file(
+    name = "make_comp_db.py",
+    src = "make_comp_db.py",
+    visibility = ["PUBLIC"],
+)
```

Comment:
Same comment as before.  None of these should be necessary.   What error do you get if you remove this?

Reply:
Unless we do this, we won't be able to use these files on our wrappers.  So this isn't needed for the copy-changes per-se, but so that we can consume them in our repo on the other pr that i posted on slack. You can't refer to files on other cells, as sources, unless they're exported into artifacts.